projects/1.16/assets/fairy-lights/fairylights/lang/zh_cn.json

```
@@ -1 +1,101 @@
-{}
\ No newline at end of file
+{
+    "itemGroup.fairylights": "Fairy Lights",

```

```suggestion
    "itemGroup.fairylights": "圣诞彩灯",
```
来自[官方语言文件](https://github.com/pau101/Fairy-Lights/blob/master/src/main/resources/assets/fairylights/lang/zh_cn.json)。

---

改动完成